api/kyverno/v2beta1/match_resources_types.go

```diff
@@ -18,6 +18,14 @@ type MatchResources struct {
 	All kyvernov1.ResourceFilters `json:"all,omitempty" yaml:"all,omitempty"`
 }
 
+// GetResourceFilters returns all resource filters
+func (m *MatchResources) GetResourceFilters() kyvernov1.ResourceFilters {
+	var filters kyvernov1.ResourceFilters
+	filters = append(filters, m.All...)
+	filters = append(filters, m.Any...)
```

Comment:
How do we differentiate `Any` and `All`?

Reply:
This function was added so that it can be used to loop through each resource filter whether it is of type `Any` or `All`.
https://github.com/kyverno/kyverno/blob/fc39bae2c781c0af988231370c090bda2b74e694/pkg/validation/cleanuppolicy/validate.go#L73-L90

In case there is a need to get either `Any` or `All`, we can access them directly.